submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+* **Per reactor, build only as many heat exchangers and steam turbines as you need.** These machines account for the majority of the UPS drain, not least because they are the ones that are built the most. This was observed by u/flame_Sla but also discussed by others, like in [this investigation](https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/) by u/warbaque.
+* **If you have long uninterrupted pipelines, replace them with lines of underground pipes.** This works because an underground pipe counts as only 2 pipes while it can replace up to 10 regular pipes. This will also increase the throughput!
+* **Limit or avoid steam storage.** While turbines and exchangers are the biggest UPS culprits and storage tanks have a relatively low impact, the difference becomes noticeable as the scale grows.
+* **Switch to solar... eventually.** In the end nuclear cannot compete with solar, which has essentially no moving parts. However, since mass-producing solar fields is a lot more expensive than nuclear, covering your initial few GW with nuclear can give you a solid start in the never-ending power problem.
+* Check: Maximize the usage of the neighbor bonus
+  * Possible resource: https://www.reddit.com/r/factorio/comments/s2h64o/100gw_nuclear_setup_ups_comparison_2x2_345ms_vs/
+* Check: Is it better to have 1 turbine less than the ratio (hence steam builds up in the system), or 1 turbine more (hence steam generally doesn't build up)
+* Check: Is it better to have more nuclear capacity than demand, or less (with the difference compensated by solar)
+* Check: Other tips?
+
+## Conclusion
+
+Hopefully this article has been helpful with demystifying nuclear power. I am glad that Wube made the mechanics so interesting to tinker with and I must praise our wonderful community full of tinkerers who have made tools and explorations, guides and videos, so that all of us can embrace a nuclear future without fear. <!--What else?-->
+
+----------
+
+
+
+<!--Does it make sense to have an appendix? Should I just post this on reddit and link it instead?-->
+
+## Appendix: Cost comparison: 40MW of solar power and accumulators vs a 40MW nuclear starter setup

```

Comment:
I'm not sure if this fits in the article :) Also, if you want to compare the setup costs you need to take down both a common platform, easy to compare. And that's typically done by taking it down to the ore. So what I would do is a simple table like this, which is small enough to fit in the article .You can then add two links to the FactorioLab pages where readers can have a closer look at the numbers.

|            	| **Solar**  	| **Nuclear** 	| **Difference** 	|
|------------	|--------	|---------	|------------	|
| Copper Ore 	| 30207  	| 5385    	| 82%        	|
| Iron Ore   	| 45320  	| 8745    	| 81%        	|
| Coal       	| 0      	| 800     	| -        	|
| Crude Oil  	| 123077 	| 16410   	| 87%        	|
| Stone      	| 0      	| 800     	| -        	|

This is easy to understand, and compare, with a quick glance :)